src/app/elections/[electionId]/results/page.tsx

```diff
@@ -1,8 +1,26 @@
+import ResultCard from "@packages/components/ResultCard";
+import candidates from "src/data/data";
```

Comment:
instead of getting the data directly, I'd create a DAO for candidates, abstracting the data access.

Therefore, when you switch your data source to a real DB, you won't need to change the data access interface

Reply:
agreed

Reply:
@Joel-leal please build this and mock until DAO level